### PR TITLE
Add write-good spell checker

### PR DESCRIPTION
**Pull request by TooAngel**
Adapt CONTRIBUTING.md and README.md accordingly.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Examples of unacceptable behavior by participants include:
 * Trolling, insulting/derogatory comments, and personal or political attacks
 * Public or private harassment
 * Publishing others' private information, such as a physical or electronic address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a professional setting
+* Other conduct which could reasonably considered inappropriate in a professional setting
```
Comment (IanFindlay) [inline CODE_OF_CONDUCT.md:23]:
```suggestion
* Other conduct which could reasonably be considered inappropriate in a professional setting
```
You need the 'be' for this sentence to make sense. Or you could reword it to something like:

"Other conduct that one could reasonably consider inappropriate in a professional setting"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 All kind of contribution is welcome, issues, contact via channels, pull requests.
 
+If you see a clear improvement, send a Pull Request.
+If you like to discuss an idea open an issue.
```
Comment (IanFindlay) [inline CONTRIBUTING.md:6]:
```suggestion
If you'd like to discuss an idea, open an issue.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
...
+If you like to talk join the discord or send an email.
```
Comment (IanFindlay) [inline CONTRIBUTING.md:7]:
```suggestion
If you'd like to talk, join the discord or send an email.
```

File: README.md [hunk 4]
```diff
@@ -81,12 +79,15 @@ enable World Driven for the repository
 - `/:org/:repo/pull/:pull_number` A detailed calculation breakdown for the pull
 request
 
-To be able to easily work on the different views test endpoints are available:
+Use the following endpoints to easier work on the dashboard with mock data:
```
Comment (IanFindlay) [inline README.md:82]:
```suggestion
Use the following endpoints to more easily work on the dashboard with mock data:
```

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (TooAngel) [reply CODE_OF_CONDUCT.md]: I reverted the change.

`write-good` was complaining about the passive voice. I'm not yet super happy with the CODE_OF_CONDUCT, it is the default one from GitHub and does not fit 100% to the World Driven idea.

File: CONTRIBUTING.md (hunk 3 above)
Comment (TooAngel) [reply CONTRIBUTING.md]: thx, fixed all of them

**Review by IanFindlay — APPROVED** [review]

**Review by IanFindlay — APPROVED** [review]

**Comment by IanFindlay** [comment]
How tied to adding this write-good are you @TooAngel ? I, more than most people, care about the proper use of language but failing CI tests due to using the passive voice or write-good mentioning that "easily can weaken meaning" seems more hassle than it is worth. I've approved the changes, as they seem fine to me, but thought I should raise the question.

**Comment by TooAngel** [comment]
> How tied to adding this write-good are you @TooAngel ? I, more than most people, care about the proper use of language but failing CI tests due to using the passive voice or write-good mentioning that "easily can weaken meaning" seems more hassle than it is worth. I've approved the changes, as they seem fine to me, but thought I should raise the question.

I'm not super tied to `write-good`, but I think having a spell checker or grammar checker helps to improve the quality of text, especially because it is not my native language. Also, I prefer that reviewers look into the Ideas behind the Pull Requests, instead of stupid spelling mistakes.

Maybe let's remove the annoying rules:
https://www.npmjs.com/package/write-good#checks

So I'm fine with removing `adverb`. Anything else where you think it is more annoying than improving the style?

**Comment by IanFindlay** [comment]
> So I'm fine with removing `adverb`. Anything else where you think it is more annoying than improving the style?

All of the rules, even the "adverb" one, seem fine to me; it's just whether the failing test is worth the improvement and how accurately it detects the defects in the first place. If it isn't too much trouble (as I see you've disabled the adverb one now and changed the sentence back) could we run it with the defaults for a while and see how it goes? The changes were an improvement and a failing test isn't as big of an issue as the red 'X' makes it seem.

**Comment by TooAngel** [comment]
K, so I would keep it like this. We can enable the adverb at some point again.

I think in an environment where you don't have a single person watching over the quality it is essential to enforce some quality metrics. So having CI checks (also for language is IMHO important), which tools to use for that is a different question and we need to get more experience.

**Comment by TooAngel** [comment]
This pull request was merged by [worlddriven](https://www.worlddriven.org).
